src/test/java/org/jenkinsci/plugins/workflow/steps/durable_task/DurableTaskStepTest.java

```diff
@@ -109,9 +109,9 @@ private static void scriptExitingAcrossRestart1(JenkinsRule r) throws Throwable
         var p = r.createProject(WorkflowJob.class, "p");
         var f = new File(r.jenkins.getRootDir(), "f");
         p.addProperty(new ParametersDefinitionProperty(new StringParameterDefinition("F")));
-        p.setDefinition(new CpsFlowDefinition("node('remote') {if (isUnix()) {sh 'sleep 5 && touch \"$F\"'} else {bat 'ping -n 5 localhost && copy nul \"%F%\" && dir \"%F%\"'}}", true));
+        p.setDefinition(new CpsFlowDefinition("node('remote') {if (isUnix()) {sh 'sleep 5 && touch \"$F\"'} else {bat 'ping -n 5 " + r.getURL().getHost() + " && copy nul \"%F%\" && dir \"%F%\"'}}", true));
```

Comment:
I guess; did you test this on Windows with IPv6?

Reply:
No, as I only had access to linux machines with IPv6 so I made sure it will not break any existing pr builder which uses ipv4 on windows and in case it breaks on windows with ipv6 for some reason we can fix later.